layers/core_validation.cpp

```diff
@@ -15313,6 +15637,7 @@ bool CoreChecks::PreCallValidateCmdBeginQueryIndexedEXT(VkCommandBuffer commandB
 
     // Extension specific VU's
     const auto &query_pool_ci = GetQueryPoolState(query_obj.pool)->createInfo;
+#if defined(VK_EXT_transform_feedback)
```

Comment:
this is inside an identical ifdef from line 15613

Reply:
Fixed in next patch